docs/particles/functional.rst

```diff
@@ -71,7 +72,7 @@ accessed using `~plasmapy.particles.standard_atomic_weight`.
 <Quantity 207.2 u>
```

Comment:
I can't directly comment on the line, but go up 3 lines from here and up the xref needs `.atomic.` added to it...

`~plasmapy.particles.atomic.standard_atomic_weight`

Reply:
Looks like there are a few others in that file. I'll learn my lesson and start a separate PR with a well-defined scope.